concrete/src/Application/Service/UserInterface.php

```diff
@@ -281,19 +282,43 @@ public function pagetabs($tabs)
     /**
      * @param $tabs
      *
+     * @param null|string $id
      * @return string
      */
-    public function tabs($tabs)
+    public function tabs($tabs, $id = null)
     {
-        $html = '<ul class="nav-tabs nav nav-fill mb-3">';
-        foreach ($tabs as $t) {
-            $dt = $t[0];
-            $href = '#' . $dt;
-            $html .= '<li class="nav-item"><a class="nav-link ' . ((isset($t[2]) && true == $t[2]) ? 'active' : '') . '" href="' . $href . '" data-toggle="tab">' . $t[1] . '</a></li>';
+        $ul = new Element("ul");
```

Comment:
When you converted this you lost the `nav-fill mb-3` classes, which make the tabs look pretty bad in most locations.

Reply:
If we're going to clean things up we need to make sure they function as they did before, or have a good reason as to why that's changed.